packages/vitest/package.json

```diff
@@ -108,6 +108,7 @@
   "dependencies": {
     "@types/chai": "^4.3.4",
     "@types/chai-subset": "^1.3.3",
+    "@types/istanbul-reports": "^3.0.1",
```

Comment:
We usually inline external package types (see jsdom). I personally don't know if we need to, we can type it as `any` for the time being if there are no better solutions. Or we can also augment types in coverage packages without installing it as the Vitest dependency.

Reply:
I see, the rollup build already inlines/bundles external typings. I'll move this into `devDependencies` instead.